### DISCUSSION TITLE
Does local clangd extension conflict with remote c/c++ extension?

### DISCUSSION THREAD

**Discussion by Xubaidu** [microsoft/vscode-cpptools#8712] [Q&A]
Hi, I am new in vscode and met a very confusing problem :(

I have successfully configured a c++ development environment in local vscode with clangd extension. (my operating system is mac os, so I used clang)

I also had to use ssh to connect my remote server with linux os. So i configured a c++ development environment in remote vscode with c/c++ extension.

However, when i work in my remote server, I found Intellisense does not work like this
![image](https://user-images.githubusercontent.com/48062706/150763080-934754bf-3c84-4af9-a0a3-103e189e2067.png)

Does the local conflict with remote? I guess not but i can not figure out what happened. Can somebody help me?

**Comment by Colengms** [comment]
Hi @Xubaidu .  This scenario should work.  Can you confirm whether this issue happens after all 'tag parsing' is complete?  Startup of IntelliSense can become blocked on successfully tag parsing of any headers it includes.  You can tell tag parsing has completed when this icon on the bottom of the VS Code window disappears: 
![image](https://user-images.githubusercontent.com/49173979/150853359-92dc66ac-daf6-462c-b5bf-6232cdb53e44.png)

Can you confirm whether you get proper IntelliSense with a very simple project?  If so, there may be some code involved in your repro that our IntelliSense parsing is failing to parse (and crashing).  If you can help us identify what code is triggering the fallback to Tag Parser, we could investigate further.

Otherwise, consider opening an issue in this repro, and providing the information requested in the new issue form.  The issue may be related to your settings and configuration.
